configurations/project.conf

```diff
@@ -0,0 +1,7 @@
+PROJECT_PREFIX="pjztlm"
```

Comment:
what is that use for ?

Reply:
the is the openshift prefix for the namespace

Reply:
coolio